src/book-demo.njk

```diff
@@ -1,8 +1,8 @@
 ---
 layout: layouts/mql-contact.njk
 title: Book a demo
-subtitle: Discover the power of FlowFuse for Node-RED projects!
-description: Book a software demo to better understand how to supercharge your workflow with high availability, enhanced data manipulation, and simplified automation. Book now for efficient Node-RED deployments!
+subtitle: Experience the fastest way to build and operate industrial data solutions
+description: See how you can streamline operations, enhance collaboration, and secure your Node-RED solutions for seamless IT/OT integration. Ready to accelerate your digital transformation? Book your demo today!
```

Comment:
Tap into our expertise to optimize operations, improve collaboration, and ensure compliance for your Node-RED solutions without any pressure or commitment.